helix-term/src/application.rs

```diff
@@ -200,40 +213,65 @@ impl Application {
             signals,
             jobs: Jobs::new(),
             lsp_progress: LspProgressMap::new(),
+            last_render: Instant::now(),
         };
 
         Ok(app)
     }
 
     fn render(&mut self) {
-        let mut cx = crate::compositor::Context {
-            editor: &mut self.editor,
-            jobs: &mut self.jobs,
-            scroll: None,
-        };
+        #[cfg(feature = "integration")]
+        return;
 
-        self.compositor.render(&mut cx);
-    }
+        #[allow(unreachable_code)]
+        {
+            let compositor = &mut self.compositor;
 
-    pub async fn event_loop(&mut self) {
-        let mut reader = EventStream::new();
-        let mut last_render = Instant::now();
-        let deadline = Duration::from_secs(1) / 60;
+            let mut cx = crate::compositor::Context {
+                editor: &mut self.editor,
+                jobs: &mut self.jobs,
+                scroll: None,
+            };
+
+            compositor.render(&mut cx);
+        }
+    }
```

Comment:
How about:

```rust
#[cfg(feature = "integration")]
fn render(&mut self) { }

#[cfg(not(feature = "integration"))]
fn render(&mut self) {
   ...
}
```

Reply:
Oh, great idea, will try

Reply:
Oh actually, it turns out because of using the test backend, it's no longer necessary to stub this out at all.